src/css/corner-layout.scss

```diff
@@ -0,0 +1,100 @@
+// Base styles
+html.easy-popup-open {
+  &:has(.easy-popup-master-corner.open) {
+    .easy-popup-master {
+      opacity: 0;
+      visibility: hidden;
+    }
+  }
+}
+
+.easy-popup-master-corner {
+  transition: opacity 0.4s ease, transform 0.4s ease;
+  transform: translateX(0);
+  position: fixed;
+  bottom: 0;
+  right: 0;
+  max-width: 450px;
```

Comment:
Please note that we are using the library for multiple purposes. So make it as flexible as you can, the `max-width` here needs to havea  CSS variable for better control.